test/acceptance/utils/StubAuthClient.scala

```diff
@@ -60,47 +60,13 @@ trait StubAuthClient {
     )
   }
 
-  private val authGGRequestJson: String =
-    """{
-      |"authorise" : [{
-      | "authProviders" : ["GovernmentGateway"]
-      |}],
-      | "retrieve" : ["allEnrolments","internalId"]
-      |}
-    """.stripMargin
-
-  def authenticateGG(eoriNumber: String): StubMapping = {
-    stubFor(post(urlEqualTo(authUrl))
-      .withRequestBody(equalToJson(authGGRequestJson))
-      .willReturn(
-        aResponse()
-          .withStatus(Status.OK)
-          .withBody(s"""{"allEnrolments": [], "internalId": "$eoriNumber"}""".stripMargin)
-      )
-    )
-  }
-
-  def authenticateGGUserAndReturnEoriEnrolment(eoriNumber: String, internalId : String): StubMapping = {
+  def authenticateGGUserWithNoEnrolments(internalId: String): StubMapping = {
     stubFor(post(urlEqualTo(authUrl))
-      .withRequestBody(equalToJson(authGGRequestJson))
+      .withRequestBody(equalToJson(authRequestJson))
       .willReturn(
         aResponse()
           .withStatus(Status.OK)
-          .withBody(
-            s"""{"allEnrolments": [
-               |  {
-               | "key": "HMRC-CUS-ORG",
-               | "identifiers": [
-               |   {
-               |     "key": "EORINumber",
-               |     "value": "$eoriNumber"
-               |   }
-               | ]
-               |}
-               |],
-               |"internalId": "$internalId"
-               |}
-              """.stripMargin)
+          .withBody(s"""{"allEnrolments": [], "internalId": "$internalId"}""".stripMargin)
```

Comment:
Is there a need for strpMargin when you keep it in one line like that?

Reply:
no, updated it.